app/controllers/mucked_homes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MuckedHomesController < ApplicationController
+
+  def index
```

Comment:
I'm not sure what the purpose of this and the VolunteersController are. Let's remove unless we really truly want them

Reply:
Okie dokie